app/models/topic.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string
```

Comment:
I think, you should add a databases constraint like not null and unique for name attribute because can the admin add a topic? or can the admin add two or more topic with the same name? probably not.

Reply:
I don't have an admin yet. But yeah, it makes sense. I thought about making it unique, but I decided not to. Not null should totally be there though.